src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,76 @@
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+/* eslint-disable no-nested-ternary */
+type Props = {
+  person: Person,
+  people: Person[];
+  handleSlugUser: (value: string) => void;
+  slugUser: string | undefined;
+  allParams: string;
+};
+
+export const PersonItem = ({
+  person,
+  people,
+  handleSlugUser,
+  slugUser,
+  allParams,
+}:Props) => {
+  return (
+    <tr
+      data-cy="person"
+      className={person.slug === slugUser
+        ? 'has-background-warning'
+        : ''}
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally. Fix all cases